v4/src/SignedIn.jsx

```diff
@@ -63,7 +63,7 @@ class SignedIn extends Component {
       guestMode,
       username,
       loadUsername,
-      selectedNode: {}, // XXX: UGLY, should use TS...
+      selectedNode: '', // XXX: UGLY, should use TS...
```

Comment:
Isn't this uglier?

Reply:
@rianadon  wanna handle migration to typescript?